english/pipboy/lockpick/lockpick_100.md

```diff
@@ -4,5 +4,7 @@ Mo' problem.
 
 Door's open. You head into the vast wasteland.
 
+[go straight until...](../love/love.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.